client/src/pages/Profile.tsx

```
@@ -316,261 +278,106 @@ const Dashboard: React.FC<DashboardProps> = ({ loggedInUsername }) => {
+                  ) : (
+                    <p>No streak data available.</p>
+                  )}
+                </CardContent>
+              </Card>
+              <Card>
+                <CardHeader>
+                  <h2 className="text-lg font-bold">LeetCode Stats</h2>
+                </CardHeader>
+                <CardContent>
+                  {leetcodeSvg ? (
+                    <div dangerouslySetInnerHTML={{ __html: leetcodeSvg }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)